media-gfx/gif2apng/gif2apng-1.9.recipe

```diff
@@ -0,0 +1,44 @@
+SUMMARY="Converts GIF animations into APNG format"
+DESCRIPTION="This program converts GIF animations into animated PNG format. \
+Usually it makes the files smaller. CLI version is OS-independent."
+HOMEPAGE="http://gif2apng.sourceforge.net/"
+COPYRIGHT="2009-2013 Max Stepin"
+LICENSE="Zlib"
+REVISION="1"
+SOURCE_URI="https://sourceforge.net/projects/gif2apng/files/$portVersion/gif2apng-$portVersion-src.zip"
+CHECKSUM_SHA256="3b21308e935d799b3ffb4a86c6e00ffa4cb9b3f72f52d58d51c66eb0574ae7d2"
+SOURCE_DIR=""
+
+ARCHITECTURES="x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
```

Comment:
If a recipe does not build on x86_gcc2 primary arch then we usually install to $prefix/bin instead of $binDir (which is $prefix/bin/x86 on x86 secondary arch).
For that purpose we define commandBinDir with these lines between SECONDARY_ARCHITECTURES and PROVIDES:
```

if [ "$targetArchitecture" != x86_gcc2 ]; then
	commandBinDir=$binDir
else
	commandBinDir=$prefix/bin
fi

```
(You can copy that block from haikuports/media-video/dvdauthor/dvdauthor-0.7.1.recipe)